File: September/Week4/Day5/teemoAttacking.py
class Solution:
    def findPoisonedDuration(self, timeSeries: List[int], duration: int) -> int:
        if len(timeSeries) == 0:
            return 0
        res = 0
        for i in range(0, len(timeSeries)-1):
            res += min(duration, timeSeries[i+1]-timeSeries[i]);
        return res + duration

File: September/Week4/Day5/teemoAttacking.c


int findPoisonedDuration(int* timeSeries, int timeSeriesSize, int duration){
    if(timeSeriesSize == 0) {
        return 0;
    }
    
    int res = 0;
    for(int i = 0; i < timeSeriesSize; i++) {
        res += fmin(duration,timeSeries[i+1]-timeSeries[i]);
    }
    
    printf(res);
}